File: themes/OrchidBsDoc/src/main/java/com/eden/orchid/bsdoc/BsDocModule.java
package com.eden.orchid.bsdoc;

import com.eden.orchid.OrchidModule;
import com.eden.orchid.api.theme.Theme;

public class BsDocModule extends OrchidModule {

    @Override
    protected void configure() {
        addToSet(Theme.class, BSDocTheme.class);
    }
}

File: themes/OrchidBsDoc/src/main/kotlin/com/eden/orchid/bsdoc/BsDocModule.kt
package com.eden.orchid.bsdoc

import com.eden.orchid.api.registration.OrchidModule
import com.eden.orchid.api.theme.Theme
import com.eden.orchid.utilities.addToSet

class BsDocModule : OrchidModule() {

    override fun configure() {
        addToSet<Theme, BSDocTheme>()
    }
}
